python/py-novaclient/Portfile

```diff
@@ -1,35 +1,42 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem        1.0
-PortGroup         python 1.0
+PortSystem          1.0
+PortGroup           python 1.0
 
-name              py-novaclient
-version           2.20.0
-categories-append net
-platforms         darwin
-maintainers       nomaintainer
-license           Apache-2
-supported_archs   noarch
+name                py-novaclient
+version             2.20.0
+revision            0
+categories-append   net
+platforms           darwin
+maintainers         nomaintainer
+license             Apache-2
+supported_archs     noarch
 
-description       Command line for openstack api
-long_description  This is a client for the OpenStack Nova API. There's a Python \
-                  API (the novaclient module), and a command-line script (nova). \
-                  Each implements 100% of the OpenStack Nova API.
-homepage          https://github.com/openstack/python-novaclient
-master_sites      pypi:p/python-novaclient/
-distname          python-novaclient-${version}
-checksums         md5 7194c91338f8583acf1b09c51d322a76 \
-                  rmd160 a8ac90ef41b75a672590eea6983cee763eff7f1f \
-                  sha256 73fc8169d58e910340aae1ac5370c3b63a9943f748e2a20e4a1dfd7277525c86
+description         Command line for openstack api
+long_description    This is a client for the OpenStack Nova API. There's a Python \
+                    API (the novaclient module), and a command-line script (nova). \
+                    Each implements 100% of the OpenStack Nova API.
+homepage            https://github.com/openstack/python-novaclient
+master_sites        pypi:p/python-novaclient/
+distname            python-novaclient-${version}
+checksums           md5 7194c91338f8583acf1b09c51d322a76 \
+                    rmd160 a8ac90ef41b75a672590eea6983cee763eff7f1f \
+                    sha256 73fc8169d58e910340aae1ac5370c3b63a9943f748e2a20e4a1dfd7277525c86
 
-python.versions   27
+python.versions     27
 
-if {$subport ne $name} {
-    depends_build    port:py${python.version}-pbr
+if {${subport} ne ${name}} {
+    depends_build-append \
+                    port:py${python.version}-pbr
 
-    depends_lib      port:py${python.version}-prettytable \
-                     port:py${python.version}-httplib2 \
-                     port:py${python.version}-simplejson \
-                     port:py${python.version}-iso8601 \
-                     port:py${python.version}-setuptools
+    depends_lib-append \
+                    port:py${python.version}-prettytable \
+                    port:py${python.version}-httplib2 \
+                    port:py${python.version}-simplejson \
+                    port:py${python.version}-iso8601 \
+                    port:py${python.version}-setuptools
```

Comment:
Since you are in here, please sort these dependencies alphabetically.

Reply:
okay, done